extensions/amp-list/0.1/amp-list.js

```diff
@@ -66,6 +97,7 @@ export class AmpList extends AMP.BaseElement {
    * @private
    */
   rendered_(elements) {
+    removeChildren(this.container_);
```

Comment:
since the list can now become very dynamic, can you please add `setAttribute('aria-live', 'polite');` to `this.element` in the `buildCallback`. That would let screen readers announce changes when list refreshes.